appserver/web/web-glue/src/main/java/com/sun/enterprise/web/WebModule.java

```diff
@@ -1178,6 +1180,20 @@ protected boolean isStandaloneModule() {
         return isStandalone;
     }
 
+    /**
+     * @return the isSystemApplication
+     */
+    public boolean isSystemApplication() {
+        return isSystemApplication;
+    }
+
+    /**
+     * @param isSystemApplication the isSystemApplication to set
```

Comment:
1. no such parameter
2. I'd drop both (here and in getter) javadocs. Noone complained about missing one for `getBean`/`setBean` for example.

Reply:
You are right, those docs don't make sense. My IDE was still configured for a project where useless docs are the norm ;)